tests/regression/test_voting_acc.py

```diff
@@ -121,3 +121,99 @@ def test_can_object(stranger, test_vote):
     # rejected vote cannot be executed
     with reverts("VOTING_CAN_NOT_EXECUTE"):
         contracts.voting.executeVote(vote_id, {"from": stranger})
+
+"""
+# stranger vote for YES holders 1,2 (reject no delegation)
+# holders 0,1 delegate to stranger
+# stranger vote for YES holder 3 (reject not a delegation for holder 3)
+# stranger vote for YES holders 1,2,3 (reject not a delegation for holder 3)
+#  stranger revote for NO holders 1,2
+# holder 3 delegate to stranger
+# stranger vote for YES holders 3
+#  holder 3 vote for NO
+#  holder 2 delegate to holder 3
+#  holder 1 reset delegate
+# stranger revote for YES holders 1-3 (reject)
+# stranger revote for YES holders 1 (reject not a delegate)
+# stranger revote for YES holders 2 (reject not a delegate)
+# stranger revote for YES holders 3 (reject cant overpower holder)
+# holders 1,2,3 can revote main phase YES
+# holders 1,2,3 can revote objection phase NO
+"""
+
+def test_attempt_vote_for_multi(call_target, stranger, test_vote):
+    vote_id = test_vote[0]
+
+    holder1 = LDO_VOTE_EXECUTORS_FOR_TESTS[0]
+    holder2 = LDO_VOTE_EXECUTORS_FOR_TESTS[1]
+    holder3 = LDO_VOTE_EXECUTORS_FOR_TESTS[3]
+
+    assert contracts.voting.getVotePhase(vote_id) == 0  # Main phase
+    # stranger vote for YES holders 1,2 (reject no delegation)
+    with reverts("REJECT"):
+        contracts.voting.attemptVoteForMultiple(vote_id, True, [holder1, holder2], {"from": stranger})
+
+    # holders 0,1 delegate to stranger
+    for holder in [holder1, holder2]:
+        contracts.voting.setDelegate(stranger, {"from": accounts.at(holder, force=True)})
```

Comment:
Semantically, it's better to introduce a standalone fixture, "delegate," cause a stranger usually means "anybody"/"entity without required rights".

Reply:
@Psirex I am not sure that we have any required for delegates at the moment, but it will add mock.